<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import Shapefile</string>
  </property>
  <widget class="QLabel" name="lblFilename">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>C:\Users\arxit\Documents\test.shp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QGIS layer :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbbLayers">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabMapping">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>581</width>
     <height>261</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="btnImport">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>File :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbbLayers</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>_selectedLayerIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImport</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>_launchImport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveConfig</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>_saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoadConfig</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>_loadConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_selectedLayerIndexChanged(int)</slot>
  <slot>_launchImport()</slot>
  <slot>_saveConfig()</slot>
  <slot>_loadConfig()</slot>
 </slots>
</ui>
